packages/asc-ui/src/components/Grid/RowStyle.ts

```diff
@@ -35,7 +36,8 @@ const RowStyle = styled.div<TypeProps>`
   display: flex;
   justify-content: ${({ halign }: { halign?: string }) => halign};
   align-items: ${({ valign }: { valign?: string }) => valign};
-  max-width: ${fromTheme('maxGridWidth')}px;
+  max-width: ${({ hasMaxWidth, theme }) =>
```

Comment:
Did you also consider overriding the theme instead of having an extra prop?

Reply:
We have some pages that will use the grid and in theory could increase in width indefinitely depending on the screen size or zoom level. Setting this in the theme to 9999999 might be an option otherwise?

Reply:
It might, but I'm not sure how easy the implementation of that use case is. The proposed solution might indeed be better.